cpu/samd5x/cpu.c

```diff
@@ -234,7 +234,7 @@ void sam0_gclk_enable(uint8_t id)
 
     switch (id) {
     case SAM0_GCLK_TIMER:
-        /* 8 MHz clock used by xtimer */
+        /* 8 MHz clock used by xtimer/ztimer_periph_timer */
```

Comment:
Unrelated to this PR.
I was wondering why my review was requested for this family.